user_tools/src/spark_rapids_pytools/cloud_api/databricks_aws.py

```diff
@@ -121,6 +122,38 @@ def pull_cluster_props_by_args(self, args: dict) -> str:
             return json.dumps(raw_prop_container.props)
         return cluster_described
 
+    def _build_ssh_cmd_prefix_for_node(self, node: ClusterNode) -> str:
```

Comment:
nit: naming convention: s/_build_ssh_cmd/_build_cmd_ssh/ to be similar to other commands?

Reply:
applies in other similar CSP files as well

Reply:
Renamed the method